Use TAILQ_LAST() instead of TAILQ_END().

The former is apparently available on OpenBSD, but not on GNU/Linux. The
calls are supposed to be equivalent, anyway.
Index: trunk/scrotwm.c
===================================================================
--- trunk.orig/scrotwm.c	2009-10-13 15:21:09.000000000 +0200
+++ trunk/scrotwm.c	2009-10-13 15:21:26.000000000 +0200
@@ -3767,7 +3767,7 @@
 	ws = win->ws;
 	wl = &ws->winlist;
 
-	for (w = TAILQ_FIRST(&ws->winlist); w != TAILQ_END(&ws->winlist); w = wn) {
+	for (w = TAILQ_FIRST(&ws->winlist); w != TAILQ_LAST(&ws->winlist, ws_win_list); w = wn) {
 		wn = TAILQ_NEXT(w, entry);
 		if (win == w)
 			continue; /* can't happen but oh well */
